strax/storage/common.py

```diff
@@ -133,21 +134,21 @@ def saver(self, key, metadata, meta_only):
 
     def get_metadata(self, key,
                      ambiguous='warn',
-                     ignore_lineage=tuple(),
-                     ignore_config=tuple()):
+                     fuzzy_for=tuple(),
```

Comment:
Does this mean that I need to update my S3 PR?  Or just raise NotImplementedError?

Reply:
Yes, if you don't have fuzzy matching implemented in a frontend, you can raise NotImplementedError whenever someone tries to use these options.

Unfortunately the best implementation of fuzzy matching is frontend specific: for filesystem directories we have to laboriously scan through directories and load jsons until we find one that matches (unless the exact match with the hash succeeds of course), for the run db we can use a mongo query.